docs/404.html

```diff
@@ -3,36 +3,36 @@
   <head>
     <meta charset="utf-8">
 <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
-<meta name="generator" content="Hugo 0.85.0" /><META NAME="ROBOTS" CONTENT="NOINDEX, NOFOLLOW">
+<meta name="generator" content="Hugo 0.84.1" /><META NAME="ROBOTS" CONTENT="NOINDEX, NOFOLLOW">
```

Comment:
Is this referring to a version of Hugo? Is there a reason we wouldn't want to use the most recent version (seems like `0.86.0`

Reply:
this is mooted by #528 build on netlify